### DISCUSSION TITLE
Application bundle value?

### DISCUSSION THREAD

**Discussion by byjrack** [microsoft/winget-cli#3190] [General]
Would like to have a first party feature in winget that would allow for groups/profiles/collections of instances in winget via manifest.  These are not explicit dependencies, but just organizational goals and operational simplification.  It's not rocket science to string together multiple winget calls in a single script, but like to get a declarative artifact and the ability to reference it via the command line and future Intune Company Portal integration.

Would this be valuable to others?

**Comment by mdanish-kh** [comment]
I'm not sure if I fully understand the request, but couldn't this be achieved with the [winget import](https://github.com/microsoft/winget-cli/blob/master/doc/windows/package-manager/winget/import.md) and [winget export](https://github.com/microsoft/winget-cli/blob/master/doc/windows/package-manager/winget/export.md) commands?
You can create a json file declaring the packages to be included in a bundle, and then let others in the organization
use `winget import` on that json file and install the packages in the bundle.

**Reply by byjrack** [reply]
No I think you got me.  I completely missed the intent of that arg.  I need to test things out, but reading the description it seems like it should work.  Some of the scoping of the installs I need to toy with, but should get me most of the way there.
